software/cloud-dashboard/react-client/src/components/temp-view/notes/notes-data.js

```diff
@@ -5,12 +5,12 @@
  */
 
 const notesData = {
-	"Kenya-Central": "Some analong inputs have been damaged. Replacement in progress.", 
-	"Kenya-North": "Some analong inputs have been damaged. Replacement in progress.",
-	"Duke": "Not in operation while a new circuit is under development.",
-	"Philippines-Sagay": "",
-	"Philippines-Subayon": "", 
-	"India": "",
+	"Kenya-Central": "Temporary decrease in location's occupancy. Outages due to biogas being conserved.", 
+	"Kenya-North": "",
+	"Duke": "Microcontroller for testing purposes only.",
+	"Philippines-Sagay": "No cellular connectivity in this region. Data transmission under development.",
+	"Philippines-Subayon": "No cellular connectivity in this region. Data transmission under development.", 
+	"India": "Heating system temporarily not it operation. Digester health being monitored.",
```

Comment:
looks like there's a small spelling mistake: "temporarily not **it** operation."